.circleci/config.yml

```diff
@@ -102,6 +102,15 @@ commands:
           command: |
             conda activate kedro-starters
             pip freeze
+      - run:
+          name: Setup Hadoop binary
+          command: |
+            Invoke-WebRequest https://github.com/steveloughran/winutils/blob/master/hadoop-2.7.1/bin/winutils.exe?raw=true -OutFile winutils.exe
```

Comment:
> Using Hadoop 2.7.1 binaries as 2.6.3 (Used on the framework) resulted in errors.

So does the test on the framework side actually works? Do we need to update this in framework too?